### PR TITLE
Resolve merge conflicts, tackled the Digital Twin Issue, updated structure (most recent version)

### PR DESCRIPTION
**Pull request by LS0205**
I resolved the merge conflicts in #20 in this branch and incoporated the issue #13 into this version, the sturcture was updated.

This version tackles the most recent issues and discussions.

### CODE REVIEW

**Comment by LS0205** [comment]
@madri2 please review this pull request.

I have added the discussed changes for this branch.

**Comment by LS0205** [comment]
@madri2 please review this PR, i have added the changes we have discussed.

File: BB-CMU/BB_Digital_Twin_proxy_in_cloud.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 
-# Digital Twin
+# Digital Twin proxy in cloud
 
 ## BB Tags(s)
 <!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
-BB-SC-TC
+BB-MU
```
Comment (madri2) [inline BB-CMU/BB_Digital_Twin_proxy_in_cloud.md:6]: @LS0205 if its a Digital Twin proxy in cloud then BB-CMU?

File: UseCases/Scenarios/BB_Template_filled_ServiceMesh_WP3.md
Comment (madri2) [inline UseCases/Scenarios/BB_Template_filled_ServiceMesh_WP3.md:1]: @LS0205 the file name need some attention? At least the WP3 should be removed.

**Review by madri2 — CHANGES_REQUESTED** [review]

File: BB-CMU/BB_Digital_Twin_proxy_in_cloud.md (hunk 1 above)
Comment (LS0205) [reply BB-CMU/BB_Digital_Twin_proxy_in_cloud.md]: done

File: UseCases/Scenarios/BB_Template_filled_ServiceMesh_WP3.md
Comment (LS0205) [reply UseCases/Scenarios/BB_Template_filled_ServiceMesh_WP3.md:1]: done

**Comment by LS0205** [comment]
@madri2, I added the requested changes.

**Review by madri2 — APPROVED** [review]
fine for me

**Pull request merged**